examples/fungible-token/tests/workspaces.rs

```diff
@@ -27,25 +27,25 @@ async fn init(
     initial_balance: U128,
 ) -> anyhow::Result<(Contract, Account, Contract)> {
     let ft_contract =
-        worker.dev_deploy(include_bytes!("../res/fungible_token.wasm").to_vec()).await?;
+        worker.dev_deploy(&include_bytes!("../res/fungible_token.wasm").to_vec()).await?;
```

Comment:
looks like you forgot to clean some of these up:
```suggestion
        worker.dev_deploy(include_bytes!("../res/fungible_token.wasm")).await?;
```